src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -22,6 +22,8 @@
 import socket
 
 from general_utils import stop_processes
+from distutils.spawn import find_executable
```

Comment:
(pylint-reimported) Reimport 'find_executable' (imported line 16)
(pylint-ungrouped-imports) Imports from package distutils are not grouped